internal/eslint-config/index.js

```diff
@@ -135,6 +135,178 @@ module.exports = defineConfig({
         'import/newline-after-import': ['error', { count: 1 }],
       },
     },
+    {
+      files: [
+        'docs/examples/**/*.{js,ts,vue}',
+        'docs/en-US/**/*.md/*.{js,ts,vue}',
+      ],
+      rules: {
+        'no-console': 'off',
+        'import/no-unresolved': 'off',
+        '@typescript-eslint/no-unused-vars': 'off',
+        'no-restricted-imports': [
+          'error',
+          {
+            paths: [
+              { name: '@element-plus', message: 'Use element-plus instead.' },
+            ],
+            patterns: [
+              {
+                group: [
+                  '@element-plus/*',
+                  '!@element-plus/icons-vue',
+                  'element-plus/es/*',
+                  '!element-plus/es/locale',
+                  'element-plus/lib/*',
+                  '!element-plus/lib/locale',
+                ],
+                message: 'Use element-plus instead.',
+              },
+            ],
+          },
+        ],
+      },
+    },
+    {
+      files: [
+        'internal/**/*.{js,ts,vue}',
```

Comment:
```suggestion
        'internal/**/*.{js,ts}',
```
There are no `.vue` files in these folders, maybe you can remove the relevant suffixes.